graph/api/src/main/java/org/jboss/windup/graph/service/GraphService.java

```diff
@@ -34,6 +34,12 @@ public GraphService(GraphContext context, Class<T> type)
         this.type = type;
     }
 
+    @SuppressWarnings("unchecked")
+    public static <T extends WindupVertexFrame> T reload(GraphContext context, T frame)
```

Comment:
Yeah, I think refresh is better. Then we can still add a reload function later (that actually reloads) without ambiguity if the need arises.